openfold3/core/model/layers/triangular_multiplicative_update.py

```diff
@@ -30,7 +30,10 @@
 from openfold3.core.model.primitives import LayerNorm, Linear
 from openfold3.core.utils.tensor_utils import permute_final_dims
 
-cueq_is_installed = importlib.util.find_spec("cuequivariance_torch") is not None
+cueq_is_installed = (
+    importlib.util.find_spec("cuequivariance_torch") is not None
+    and torch.cuda.is_available()
+)
```

Comment:
Can we mutualize this into a helper function? it seems to be duplicated 2-3 times in this PR